providers/renegade-provider/config.ts

```diff
@@ -20,6 +19,5 @@ export const getConfigFromChainId = (chainId: number) => {
     storage: createStorage({
       storage: cookieStorage,
     }),
-    viemClient: viemClient as any,
```

Comment:
is it ok to remove this? we rely on `viemClient` when we fetch logs to see if we should lookup a wallet or not. if unset the config will default to using an arbitrum sepolia public client